### PR TITLE
[es] Remove {{ARIARole}} macro

### PR DESCRIPTION
**Pull request by rwxdan**
### Description

- Remove deprecated `{{AriaRole}}` macro from `l10n-es`

### Related issues and pull requests

- #11185

### CODE REVIEW

File: files/es/web/html/element/header/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ El _elemento de HTML Header_ (\<header>) representa un grupo de ayudas introduct
...
-| Elementos padres permitidosRoles ARIA permitidosInreface DOM | Cualquier elemento que acepte [contenido dinámico](/en/HTML/Content_categories#flow_content). Ten en cuenta que un elemento `<header>` no debe ser descendiente de un elemento {{ HTMLElement("address") }}, {{ HTMLElement("footer") }} o cualquier otro elemento {{ HTMLElement("header") }}.{{ARIARole("group")}}, {{ARIARole("presentation")}}{{domxref("HTMLElement")}} |
+| Elementos padres permitidosRoles ARIA permitidosInreface DOM | Cualquier elemento que acepte [contenido dinámico](/en/HTML/Content_categories#flow_content). Ten en cuenta que un elemento `<header>` no debe ser descendiente de un elemento {{ HTMLElement("address") }}, {{ HTMLElement("footer") }} o cualquier otro elemento {{ HTMLElement("header") }}.<a href="/es/docs/Web/Accessibility/ARIA/Roles/group_role"><code>group</code></a>, <a href="/es/docs/Web/Accessibility/ARIA/Roles/presentation_role"><code>presentation</code></a>{{domxref("HTMLElement")}} |
```
Comment (yin1999) [inline files/es/web/html/element/header/index.md:18]:
```suggestion
| Elementos padres permitidosRoles ARIA permitidosInreface DOM | Cualquier elemento que acepte [contenido dinámico](/en/HTML/Content_categories#flow_content). Ten en cuenta que un elemento `<header>` no debe ser descendiente de un elemento {{ HTMLElement("address") }}, {{ HTMLElement("footer") }} o cualquier otro elemento {{ HTMLElement("header") }}. [`group`](/es/docs/Web/Accessibility/ARIA/Roles/group_role), [`presentation`](/es/docs/Web/Accessibility/ARIA/Roles/presentation_role) {{domxref("HTMLElement")}} |
```

**Review by yin1999 — COMMENTED** [review]
In pure markdown, please use GFM format :)

File: files/es/web/html/element/header/index.md (hunk 1 above)
Comment (rwxdan) [reply files/es/web/html/element/header/index.md:18]: Noted. Thank you

File: files/es/web/html/element/header/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ El _elemento de HTML Header_ (\<header>) representa un grupo de ayudas introduct
...
-| Elementos padres permitidosRoles ARIA permitidosInreface DOM | Cualquier elemento que acepte [contenido dinámico](/en/HTML/Content_categories#flow_content). Ten en cuenta que un elemento `<header>` no debe ser descendiente de un elemento {{ HTMLElement("address") }}, {{ HTMLElement("footer") }} o cualquier otro elemento {{ HTMLElement("header") }}.{{ARIARole("group")}}, {{ARIARole("presentation")}}{{domxref("HTMLElement")}} |
+| Elementos padres permitidosRoles ARIA permitidosInreface DOM | Cualquier elemento que acepte [contenido dinámico](/en/HTML/Content_categories#flow_content). Ten en cuenta que un elemento `<header>` no debe ser descendiente de un elemento {{ HTMLElement("address") }}, {{ HTMLElement("footer") }} o cualquier otro elemento {{ HTMLElement("header") }}. [`group`](/es/docs/Web/Accessibility/ARIA/Roles/group_role), [`presentation`](/es/docs/Web/Accessibility/ARIA/Roles/presentation_role) {{domxref("HTMLElement")}} |
```
Comment (Graywolf9) [inline files/es/web/html/element/header/index.md:18]: Fix format error from origin

```suggestion
| Elementos padres permitidos | Cualquier elemento que acepte [contenido dinámico](/en/HTML/Content_categories#flow_content). Ten en cuenta que un elemento `<header>` no debe ser descendiente de un elemento {{ HTMLElement("address") }}, {{ HTMLElement("footer") }} o cualquier otro elemento {{ HTMLElement("header") }}. |
| Roles ARIA permitidos | [`group`](/es/docs/Web/Accessibility/ARIA/Roles/group_role), [`presentation`](/es/docs/Web/Accessibility/ARIA/Roles/presentation_role) |
| Interfaz DOM | {{domxref("HTMLElement")}} |
```

File: files/es/web/html/element/object/index.md [hunk 3]
```diff
@@ -71,8 +71,8 @@ La fuente original de este ejemplo interactivo está almacenada en un repositori
     <tr>
       <th scope="row">Permitted ARIA roles</th>
       <td>
-        {{ARIARole("application")}}, {{ARIARole("document")}},
-        {{ARIARole("image")}}
+        <a href="/es/docs/Web/Accessibility/ARIA/Roles/application_role"><code>application</code></a>, <a href="/es/docs/Web/Accessibility/ARIA/Roles/document_role"><code>document</code></a>,
+        <a href="/es/docs/Web/Accessibility/ARIA/Roles/image_role"><code>image</code></a>
```
Comment (Graywolf9) [inline files/es/web/html/element/object/index.md:75]: It maybe should be `img` as the file above, suggesting the change to in `en-US` in https://github.com/mdn/content/pull/23869
```suggestion
        <a href="/es/docs/Web/Accessibility/ARIA/Roles/img_role"><code>img</code></a>
```

**Review by Graywolf9 — APPROVED** [review]
Excellent work! Thank you so much! Feel free to keep contributing!

**Pull request merged**